docs/appendix/bill-of-materials.md

```diff
@@ -67,14 +67,16 @@ The following materials are distributed with Zowe&trade; binaries:
 | scripts/zss/zowe-xmem-ppt.sh | Script to read Program Properties Table for Zowe ZSS Server |  [zowe-install-packaging/scripts/zss](https://github.com/zowe/zowe-install-packaging/tree/master/scripts/zss) | [install-packaging pipeline](https://github.com/zowe/zowe-install-packaging/blob/master/Jenkinsfile) |
 
-## Zowe CLI Package  
+## Zowe CLI Package
 
-| File Path | Description | Source Location | Build Script(s) | 
+| File Path | Description | Source Location | Build Script(s) |
 | --------- | ----------- | --------------- | --------------- |
-| <>/zowe-cli-package-*.zip | Zowe CLI package containing the core Zowe CLI as well as Zowe CLI Plugins | Multiple open source repositories within [zowe](https://www.github.com/zowe) | [zowe-cli-standalone-package pipeline](https://github.com/zowe/zowe-cli-standalone-package/blob/master/Jenkinsfile) |  
-| licenses/zowe_licenses_full.zip | License archive for Zowe third party dependencies | n/a | Build coming soon |  
+| <>/zowe-cli-package-*.zip | Zowe CLI package containing the core Zowe CLI as well as Zowe CLI Plugins | Multiple open source repositories within [zowe](https://www.github.com/zowe) | [zowe-cli-standalone-package pipeline](https://github.com/zowe/zowe-cli-standalone-package/blob/master/Jenkinsfile) |
+| licenses/zowe_licenses_full.zip | License archive for Zowe third party dependencies | n/a | Build coming soon |
 | zowe-cli-*.tgz | Zowe CLI | [zowe-cli](https://github.com/zowe/zowe-cli/) | [zowe-cli pipeline](https://github.com/zowe/zowe-cli/blob/master/Jenkinsfile) |
-| zowe-cics-*.tgz |  IBM® CICS® Plug-in for Zowe CLI | [zowe-cli-cics-plugin](https://github.com/zowe/zowe-cli-cics-plugin) | [zowe-cli-cics-plugin pipeline](https://github.com/zowe/zowe-cli-cics-plugin/blob/master/Jenkinsfile) | 
-| zowe-db2-*.tgz | Zowe CLI Plug-in for IBM® Db2® for z/OS® | [zowe-cli-db2-plugin](https://github.com/zowe/zowe-cli-db2-plugin) | [zowe-cli-db2-plugin pipeline](https://github.com/zowe/zowe-cli-db2-plugin/blob/master/Jenkinsfile) | 
+| zowe-cics-*.tgz |  IBM® CICS® Plug-in for Zowe CLI | [zowe-cli-cics-plugin](https://github.com/zowe/zowe-cli-cics-plugin) | [zowe-cli-cics-plugin pipeline](https://github.com/zowe/zowe-cli-cics-plugin/blob/master/Jenkinsfile) |
+| zowe-db2-*.tgz | Zowe CLI Plug-in for IBM® Db2® for z/OS® | [zowe-cli-db2-plugin](https://github.com/zowe/zowe-cli-db2-plugin) | [zowe-cli-db2-plugin pipeline](https://github.com/zowe/zowe-cli-db2-plugin/blob/master/Jenkinsfile) |
 | zowe-ims-*.tgz | IBM® IMS™ Plug-in for Zowe CLI | [zowe-cli-mq-plugin](https://github.com/zowe/zowe-cli-mq-plugin) | [zowe-cli-mq-plugin](https://github.com/zowe/zowe-cli-mq-plugin/blob/master/Jenkinsfile) |
```

Comment:
It looks like some of these rows are misaligned; i.e. zowe-scs shows an ims pipeline text, zowe-ims shows an mq plugin

Reply:
my mistake, will resolve today!